miner/worker.go

```diff
@@ -269,6 +335,35 @@ func (w *worker) newWorkLoop() {
 				recommit(true, commitInterruptResubmit)
 			}
 
+		case interval := <-w.resubmitIntervalCh:
+			// Adjust resubmit interval explicitly by user.
+			if interval < minRecommitInterval {
+				log.Info("Recap miner recommit interval", "from", interval, "to", minRecommitInterval)
```

Comment:
Lets call this Sanitizing instead of Recap, it's clearer and also what we use in txpool. Instead of from/to please use provided and updated. Also please raise the log level from INFO to WARN. It's definitely an error, even though we can handle it.